crates/ruff_python_formatter/tests/snapshots/format@fmt_skip__compound_one_liners.py.snap

```diff
@@ -0,0 +1,289 @@
+---
+source: crates/ruff_python_formatter/tests/fixtures.rs
+input_file: crates/ruff_python_formatter/resources/test/fixtures/ruff/fmt_skip/compound_one_liners.py
+---
+## Input
+```python
+# Test cases for fmt: skip on compound statements that fit on one line
+
+# Basic single-line compound statements
+def simple_func(): return "hello"  # fmt: skip
+if True: print("condition met")  # fmt: skip
+for i in range(5): print(i)  # fmt: skip
+while x < 10: x += 1  # fmt: skip
+
+# With expressions that would normally trigger formatting
+def long_params(a, b, c, d, e, f, g): return a + b + c + d + e + f + g  # fmt: skip
+if some_very_long_condition_that_might_wrap: do_something_else_that_is_long()  # fmt: skip
+
+# Nested compound statements (outer should be preserved)
+if True:
+    for i in range(10): print(i)  # fmt: skip
+
+# Multiple statements in body (should not apply - multiline)
+if True:
+    x = 1
+    y = 2  # fmt: skip
```

Comment:
Change it to something that would format differently to demonstrate that this doesn't break normal `fmt: skip` behavior
```suggestion
    y  =  2  # fmt: skip
```